rankers/SimpleAggregateRanker/manifest.yml

```diff
@@ -0,0 +1,13 @@
+manifest_version: 1
+name: SimpleAggregateRanker
+kind: ranker
+type: pod
+description: :class:`SimpleAggregateRanker` calculates the score of the matched doc from the matched chunks.
```

Comment:
`SimpleAggregateRanker` combines (or reduces?) the score of several documents into one unique score. It is used to aggregate the chunk scores into parent document scores

Reply:
Aggregate, reduce, accumulate or compress are all synonyms in functional programming. So I have no preference. But I wouldn't use `combine`

Reply:
yes, good point, anyway is good

Reply:
changed it to `aggregates`